src/sentry/static/sentry/app/components/events/interfaces/contextLine.jsx

```diff
@@ -18,11 +19,22 @@ const ContextLine = function(props) {
   }
 
   return (
-    <li className={liClassName} key={line[0]}>
-      <span className="ws">{lineWs}</span>
-      <span className="contextline">{lineCode}</span>
-      {isActive && <OpenInButton filename={filename} lineNo={line[0]} group={group} />}
-    </li>
+    <React.Fragment>
+      {isActive ? (
+        <OpenInButton
+          group={group}
+          filename={filename}
+          lineNo={line[0]}
+          lineWs={lineWs}
+          lineCode={lineCode}
+        />
+      ) : (
+        <ListItem className={liClassName} key={line[0]}>
+          <span className="ws">{lineWs}</span>
+          <span className="contextline">{lineCode}</span>
+        </ListItem>
+      )}
+    </React.Fragment>
```

Comment:
Where is `ContextLine` used? How is `isActive` passed and why does that dictate whether we render the `OpenInButton`?

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/events/interfaces/frame.jsx#L270

Reply:
we only ever render the `OpenInButton` when its an active line - we don't add it for all the other lines in the frame